tfjs-backend-webgpu/src/kernels/FromPixelsExternalImage.ts

```diff
@@ -16,10 +16,62 @@
  */
 
 import {FromPixelsAttrs, TensorInfo, util} from '@tensorflow/tfjs-core';
+
 import {WebGPUBackend} from '../backend_webgpu';
+
+import {FromPixelsImportProgram} from './FromPixels_utils/from_pixels_import_webgpu';
 import {FromPixelsProgram} from './FromPixels_utils/from_pixels_webgpu';
 import * as webgpu_program from './webgpu_program';
 
+export function fromPixelsImportTexture(args: {
```

Comment:
Is it possible to merge `fromPixelsImportTexture` and `fromPixelsExternalImage` and you can use the program as the parameter of this function?